tds-item-selection-components/src/main/java/tds/itemselection/model/AlgorithmType.java

```diff
@@ -0,0 +1,34 @@
+package tds.itemselection.model;
+
+/**
+ * The available algorithm types that are supported by item selection
+ */
+public enum AlgorithmType {
```

Comment:
This is very much like the algorithm type we have in our new code.  However, I didn't want to start adding our new code dependencies to these legacy projects.